Documentation/profiles.md

```diff
@@ -53,7 +53,18 @@ TODO
 
 ### Docker with User Namespaces
 
-TODO
+This profile allows user to start container with support of user namespace.
+
+Differences from the default docker profile:
+
+* The processes in the container run as root of the user namespace of containers.
+
+* User mapping is done in `/proc/self/uid_map` as following:
+
+```
+0     100000      65536
+```
+
```

Comment:
The following kinds of namespaces are owned by the user namespace of the container:
- ipc
- mnt
- net
- pid
- uts

The following kinds of namespaces are owned by the root user namespace:
- user
- cgroup

Reply:
done